server/athenian/api/application.py

```diff
@@ -1018,3 +1022,19 @@ def _set_unready(self) -> None:
         if (health := self.app.get("health")) is not None:
             self.app[PROMETHEUS_REGISTRY_VAR_NAME].unregister(health)
             del self.app["health"]
+
+
+@functools.cache
+def _get_ref_resolver_store() -> dict[str, dict]:
+    """Build the custom ref resolver store for especifico.
+
+    This store avoids network access to resolve references to public API schema.
+
+    """
+    public_schema_url = "https://api.athenian.co/v1/openapi.json"
+    public_schema_path = Path(athenian.api.__file__).parent / "openapi" / "openapi.yaml"
+
+    with open(public_schema_path, "r") as fp:
+        public_schema = yaml.load(fp, Loader=yaml.CSafeLoader)
+
+    return {public_schema_url: public_schema}
```

Comment:
Let's have a trivial classmethod `def (cls, ...)` with a manually managed cache in a class-level variable, it will be a simpler code and easier to debug.